backend/main.go

```diff
@@ -270,5 +278,8 @@ func categorizeItems(orderHistory *[]OrderInfo) ItemCategories {

```

Comment:
i would comment this line out, but otherwise it's good :) 

aka i think the code should be able to cloned and be ready to go for lambda, and if you want it to run locally, then its just commenting out one line and commenting in the other